### PR TITLE
Logo updates removing feedback link

### PR DESCRIPTION
**Pull request by nnunn**
Updating ualberta and gp logos. Removing feedback link as this is no longer monitored.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ and releases in NEOSDiscovery project adheres to [Semantic Versioning](http://se
 ### Changed
 - log level from debug to warn [PR#412](https://github.com/ualbertalib/NEOSDiscovery/pull/412)
 
+## [1.0.63] - 2020-09-21
```
Comment (pgwillia) [inline CHANGELOG.md:12]: There's a "Changed" item in the unreleased section that should go with these ones too.

**Review by pgwillia — CHANGES_REQUESTED** [review]
For preparing for a release there's one more place to bump the version number: https://github.com/ualbertalib/NEOSDiscovery/blob/1aad9fd34d1d30e934b80669209a42681b6fe3ee/config/application.rb#L10
Then we can check the version deployed by viewing the html source
image

I had a comment about combining the Unreleased section.  Then this looks good to go!

**Comment by nnunn** [comment]
I've made those changes - thanks!

**Review by pgwillia — APPROVED** [review]
You can go ahead with the release too, unless you want me to.

**Comment by nnunn** [comment]
I can -- thanks!

**Pull request merged**